github-orgs/kubeflow/org.yaml

```diff
@@ -1010,6 +1010,22 @@ orgs:
             privacy: closed
             repos:
               katib: write
+          wg-data-leads:
+            description: Team of Model Registry and Spark Operator
+            members:
+            - andreyvelich
+            - ckadner
+            - dhirajsb
+            - isinyaaa
+            - lampajr
+            - rareddy
+            - rkubis
+            - tarilabs
+            - Tomcli
+            - tonyxrmdavidson
+            privacy: closed
+            repos:
+              model-registry: write
```

Comment:
This repo doesn't exist yet. Please remove for now and submit another PR later to add this back. Otherwise internal-acls pipeline will be broken.